src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -951,34 +961,23 @@ void GrpcLb::BalancerCallState::StartQuery() {
 }
 
 void GrpcLb::BalancerCallState::ScheduleNextClientLoadReportLocked() {
-  // InvalidateNow to avoid getting stuck re-initializing this timer
-  // in a loop while draining the currently-held WorkSerializer.
-  // Also see https://github.com/grpc/grpc/issues/26079.
-  ExecCtx::Get()->InvalidateNow();
-  const Timestamp next_client_load_report_time =
-      ExecCtx::Get()->Now() + client_stats_report_interval_;
-  GRPC_CLOSURE_INIT(&client_load_report_closure_, MaybeSendClientLoadReport,
-                    this, grpc_schedule_on_exec_ctx);
-  grpc_timer_init(&client_load_report_timer_, next_client_load_report_time,
-                  &client_load_report_closure_);
-  client_load_report_timer_callback_pending_ = true;
+  client_load_report_handle_ = GetDefaultEventEngine()->RunAt(
+      absl::Now() + absl::Milliseconds(client_stats_report_interval_.millis()),
+      absl::bind_front(&GrpcLb::BalancerCallState::MaybeSendClientLoadReport,
```

Comment:
Is there a reason to prefer `absl::bind_front` over a direct lambda: `[this] { MaybeSendClientLoadReport(); }`

(I prefer the latter because less combinators, and I think it leads to less edits as requirements change)

Reply:
Change applied, thanks. There was no great reason for bind_front.